renovate.json

```diff
@@ -400,6 +400,32 @@
       "enabled": true,
       "minimumReleaseAge": "7 days"
     },
+    {
+      "groupName": "@elastic/kibana-management dependencies",
+      "matchDepNames": [
+        "@elastic/request-converter",
+        "@mapbox/vector-tile",
+        "cronstrue",
+        "file-saver",
```

Comment:
This one is also used quite a bit by the canvas plugin, should we separate it and add both teams as reviewers?

Reply:
`file-saver` ownership was discussed in the ownership report, and @mattkime confirmed ownership for the management team.

> Canvas is targeted for removal in 10.0. If management continues to use this dependency presentation team would suggest management take ownership.

Reply:
Got it, wasnt aware of that conversation